modules/builds-configmaps-use-case-consuming-in-volumes.adoc

```diff
@@ -0,0 +1,85 @@
+// Module included in the following assemblies:
+//
+//* builds/builds-configmaps.adoc
+
+[id="builds-configmaps-use-case-consuming-in-volumes_{context}"]
+= Consuming ConfigMaps in volumes
```

Comment:
It might be my limited understanding as to what is happening in this procedure, but I am finding it difficult to determine what is meant by 'consuming ConfigMaps in volumes'. This was the same for pods. I think it might be more useful for the reader if this title and subsequent references in this module are more explicit. Would it be more precise to say something like this, or have I misunderstood the mechanism?:

=  Injecting content into a volume by using ConfigMaps

Please ignore me if I have just misunderstood :-)